__tests__/get.test.js

```diff
@@ -65,6 +65,27 @@ describe("GET /api/reviews", () => {
 
 describe("GET /api/reviews/:review_id", () => {
   test("should return all the reviews with a given id", () => {
+    return request(app)
+      .get("/api/reviews/2")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toBeInstanceOf(Array);
+        expect(body.review.length).toBeGreaterThan(0);
+        expect(body.review[0]).toEqual({
```

Comment:
You shouldn't be altering passing tests, ideally. I'd recommend using .toMatchObject rather than .toEqual in future as it won't fail if a property is added.